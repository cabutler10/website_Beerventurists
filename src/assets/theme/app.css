html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
