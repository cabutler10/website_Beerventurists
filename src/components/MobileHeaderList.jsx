import React from 'react'
import PropTypes from 'prop-types'
import Link from './Link'
import { makeStyles } from '@material-ui/styles'
import Button from '@material-ui/core/Button'
import List from '@material-ui/core/List'
import ListItem from '@material-ui/core/ListItem'
import ListItemText from '@material-ui/core/ListItemText'
import ArrowIcon from '@material-ui/icons/KeyboardArrowRight'
import FacebookIcon from '../assets/icons/Facebook'
import InstagramIcon from '../assets/icons/Instagram'

const useStyles = makeStyles(theme => ({
  list: {
    width: 200,
  },
  text: {
    textTransform: 'uppercase',
    paddingTop: 15,
    paddingBottom: 15,
    paddingLeft: 30,
    paddingRight: 30,
  },
  link: {
    color: theme.status.greyDk,
  },
  icon: {
    color: theme.status.grey,
    marginRight: 15,
  },
  button: {
    width: '100%',
    padding: 0,
  },
  iconSocial: {
    width: 18,
    height: 18,
    fill: theme.status.black,
  },
}))

function MobileHeaderList({ links, linkLabels, lng, changeLng }) {
  const classes = useStyles()
  return (
    <div className={classes.list}>
      <List component="nav">
        <Link to={`/`}>
          <ListItem button divider dense>
            <ListItemText
              disableTypography
              className={classes.text}
              primary="home"
            />
            <ArrowIcon className={classes.icon} />
          </ListItem>
        </Link>
        <Button
          aria-label="shop"
          className={classes.button}
          href="https://shop.eisbach-riders.com/shop/"
          disableFocusRipple
          disableRipple
        >
          <ListItem button divider dense>
            <ListItemText
              disableTypography
              className={classes.text}
              primary={linkLabels[0]}
            />
            <ArrowIcon className={classes.icon} />
          </ListItem>
        </Button>
        <Button
          aria-label="facebook"
          className={classes.button}
          href="https://www.facebook.com/EisbachRiders/"
          target="_blank"
          rel="noopener"
          disableFocusRipple
          disableRipple
        >
          <ListItem button divider dense>
            <ListItemText
              disableTypography
              className={classes.text}
              primary={<FacebookIcon className={classes.iconSocial} />}
            />

            <ArrowIcon className={classes.icon} />
          </ListItem>
        </Button>
        <Button
          aria-label="instagram"
          className={classes.button}
          href="https://www.instagram.com/eisbachriders/"
          target="_blank"
          rel="noopener"
          disableFocusRipple
          disableRipple
        >
          <ListItem button divider dense>
            <ListItemText
              disableTypography
              className={classes.text}
              primary={<InstagramIcon className={classes.iconSocial} />}
            />
            <ArrowIcon className={classes.icon} />
          </ListItem>
        </Button>
        <Button
          aria-label="change language"
          className={classes.button}
          onClick={changeLng}
        >
          <ListItem button divider dense>
            <ListItemText
              disableTypography
              className={classes.text}
              primary={lng === 'en' ? 'de' : 'en'}
            />
            <ArrowIcon className={classes.icon} />
          </ListItem>
        </Button>
      </List>
    </div>
  )
}

MobileHeaderList.propTypes = {
  links: PropTypes.array.isRequired,
  linkLabels: PropTypes.array.isRequired,
  lng: PropTypes.string.isRequired,
  changeLng: PropTypes.func.isRequired,
}

export default MobileHeaderList
