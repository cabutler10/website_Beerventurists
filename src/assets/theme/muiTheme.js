import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
