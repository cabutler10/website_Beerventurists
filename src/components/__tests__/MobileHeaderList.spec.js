import React from 'react'
import { render } from 'react-testing-library'
import MobileHeaderList from '../MobileHeaderList'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'

describe('MobileHeaderList', () => {
  const mockProps = {
    links: [],
    linkLabels: [],
  }
  it('renders correctly', () => {
    const component = render(
      <ThemeProvider theme={theme}>
        <MobileHeaderList {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
