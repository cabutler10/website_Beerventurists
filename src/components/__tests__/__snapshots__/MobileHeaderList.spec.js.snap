// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileHeaderList renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-list-1fy7rlu"
      >
        <nav
          class="MuiList-root-1 MuiList-padding-2"
        >
          <a
            class="MuiTypography-root-10 MuiTypography-colorPrimary-40 MuiTypography-inline-45 MuiLink-root-5 MuiLink-underlineHover-7"
            href="/"
            style="text-decoration: none;"
          >
            <div
              class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
              >
                home
              </div>
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
                <path
                  d="M0 0h24v24H0V0z"
                  fill="none"
                />
              </svg>
              <span
                class="MuiTouchRipple-root-102"
              />
            </div>
          </a>
          <a
            aria-label="shop"
            class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
            href="https://shop.eisbach-riders.com/shop/"
            role="button"
            tabindex="0"
          >
            <span
              class="MuiButton-label-77"
            >
              <div
                class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                  />
                  <path
                    d="M0 0h24v24H0V0z"
                    fill="none"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root-102"
                />
              </div>
            </span>
          </a>
          <a
            aria-label="facebook"
            class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
            href="https://www.facebook.com/EisbachRiders/"
            rel="noopener"
            role="button"
            tabindex="0"
            target="_blank"
          >
            <span
              class="MuiButton-label-77"
            >
              <div
                class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
                >
                  <svg
                    aria-labelledby="title"
                    class="Hook-iconSocial-1w0l16t"
                    height="32"
                    viewBox="0 0 96.124 96.123"
                    width="32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <title
                      id="title_facebook"
                    >
                      Facebook Icon
                    </title>
                    <path
                      d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803 c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654 c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246 c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                  />
                  <path
                    d="M0 0h24v24H0V0z"
                    fill="none"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root-102"
                />
              </div>
            </span>
          </a>
          <a
            aria-label="instagram"
            class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
            href="https://www.instagram.com/eisbachriders/"
            rel="noopener"
            role="button"
            tabindex="0"
            target="_blank"
          >
            <span
              class="MuiButton-label-77"
            >
              <div
                class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
                >
                  <svg
                    aria-labelledby="title"
                    class="Hook-iconSocial-1w0l16t"
                    height="32"
                    viewBox="0 0 448 512"
                    width="32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <title
                      id="title_instagram"
                    >
                      Instagram Icon
                    </title>
                    <path
                      d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                  />
                  <path
                    d="M0 0h24v24H0V0z"
                    fill="none"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root-102"
                />
              </div>
            </span>
          </a>
          <button
            aria-label="change language"
            class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label-77"
            >
              <div
                class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
                >
                  en
                </div>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                  />
                  <path
                    d="M0 0h24v24H0V0z"
                    fill="none"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root-102"
                />
              </div>
            </span>
            <span
              class="MuiTouchRipple-root-102"
            />
          </button>
        </nav>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-list-1fy7rlu"
    >
      <nav
        class="MuiList-root-1 MuiList-padding-2"
      >
        <a
          class="MuiTypography-root-10 MuiTypography-colorPrimary-40 MuiTypography-inline-45 MuiLink-root-5 MuiLink-underlineHover-7"
          href="/"
          style="text-decoration: none;"
        >
          <div
            class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
            role="button"
            tabindex="0"
          >
            <div
              class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
            >
              home
            </div>
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root-67 Hook-icon-evip3s"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
              <path
                d="M0 0h24v24H0V0z"
                fill="none"
              />
            </svg>
            <span
              class="MuiTouchRipple-root-102"
            />
          </div>
        </a>
        <a
          aria-label="shop"
          class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
          href="https://shop.eisbach-riders.com/shop/"
          role="button"
          tabindex="0"
        >
          <span
            class="MuiButton-label-77"
          >
            <div
              class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
                <path
                  d="M0 0h24v24H0V0z"
                  fill="none"
                />
              </svg>
              <span
                class="MuiTouchRipple-root-102"
              />
            </div>
          </span>
        </a>
        <a
          aria-label="facebook"
          class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
          href="https://www.facebook.com/EisbachRiders/"
          rel="noopener"
          role="button"
          tabindex="0"
          target="_blank"
        >
          <span
            class="MuiButton-label-77"
          >
            <div
              class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
              >
                <svg
                  aria-labelledby="title"
                  class="Hook-iconSocial-1w0l16t"
                  height="32"
                  viewBox="0 0 96.124 96.123"
                  width="32"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title_facebook"
                  >
                    Facebook Icon
                  </title>
                  <path
                    d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803 c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654 c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246 c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
                <path
                  d="M0 0h24v24H0V0z"
                  fill="none"
                />
              </svg>
              <span
                class="MuiTouchRipple-root-102"
              />
            </div>
          </span>
        </a>
        <a
          aria-label="instagram"
          class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
          href="https://www.instagram.com/eisbachriders/"
          rel="noopener"
          role="button"
          tabindex="0"
          target="_blank"
        >
          <span
            class="MuiButton-label-77"
          >
            <div
              class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
              >
                <svg
                  aria-labelledby="title"
                  class="Hook-iconSocial-1w0l16t"
                  height="32"
                  viewBox="0 0 448 512"
                  width="32"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title_instagram"
                  >
                    Instagram Icon
                  </title>
                  <path
                    d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
                <path
                  d="M0 0h24v24H0V0z"
                  fill="none"
                />
              </svg>
              <span
                class="MuiTouchRipple-root-102"
              />
            </div>
          </span>
        </a>
        <button
          aria-label="change language"
          class="MuiButtonBase-root-58 MuiButton-root-76 MuiButton-text-78 MuiButton-flat-81 Hook-button-44tu7v"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label-77"
          >
            <div
              class="MuiButtonBase-root-58 MuiListItem-root-46 MuiListItem-default-49 MuiListItem-dense-50 MuiListItem-gutters-54 MuiListItem-divider-53 MuiListItem-button-55"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemText-root-61 MuiListItemText-dense-63 Hook-text-q4cult"
              >
                en
              </div>
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root-67 Hook-icon-evip3s"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
                <path
                  d="M0 0h24v24H0V0z"
                  fill="none"
                />
              </svg>
              <span
                class="MuiTouchRipple-root-102"
              />
            </div>
          </span>
          <span
            class="MuiTouchRipple-root-102"
          />
        </button>
      </nav>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
